modules/auxiliary/admin/postgres/postgres_sql.rb

```diff
@@ -18,10 +19,17 @@ def initialize(info = {})
       'References'     =>
         [
           [ 'URL', 'www.postgresql.org' ]
-        ]
+        ],
+      'SessionTypes' => %w[PostgreSQL]
     ))
+  end
+
+  def rhost
```

Comment:
Can we put this back to where you found it? 😁 

And let's squash your commits down into a collection of commits that makes sense (probably 1?)